src/java/org/apache/cassandra/utils/FBUtilities.java

```diff
@@ -520,6 +520,22 @@ public static IAuditLogger newAuditLogger(String className) throws Configuration
         return FBUtilities.construct(className, "Audit logger");
     }
 
+    public static boolean isAuditLoggerClassExists(String className)
```

Comment:
maybe can you re-use ``FBUtilities.construct`` rather than this method?

Reply:
`FBUtilities.construct` instantiates an object of a class, hence added this new method